configuration/analyzer_config.json

```diff
@@ -902,6 +902,14 @@
       "api_key_name": "VT_KEY"
     }
   },
+  "Whoisxmlapi": {
+    "type": "observable",
+    "observable_supported": ["ip", "domain"],
+    "external_service": true,
+    "description": "the WHOIS record data, of a domain name, an IP address, or an email address",
+    "requires_configuration": true,
+    "python_module": "whoisxmlapi_run"
```

Comment:
```
 "additional_config_params": {
      "api_key_name": "WHOISXMLAPI_KEY"
 }
```

Add this entry too just after `python_module`.

Reply:
I just change and build new Intel Owl. However, result is no change. This is log:
```
2020-09-07 07:01:00 - api_app.crons - flush_expired_tokens - INFO - started flush_expired_tokens
2020-09-07 07:01:00 - api_app.crons - flush_expired_tokens - INFO - finished flush_expired_tokens
2020-09-07 07:01:25 - api_app.auth - perform_logout - INFO - perform_logout received request from root.
2020-09-07 07:01:25 - api_app.auth - perform_logout - INFO - administrator:'root' was logged out.
2020-09-07 07:01:25 - api_app.auth - perform_logout - INFO - perform_logout: current token was blacklisted.
2020-09-07 07:01:27 - api_app.auth - obtain_user_token - INFO - obtain_user_token received request for 'root'.
2020-09-07 07:01:28 - api_app.auth - obtain_user_token - INFO - headers: {'Content-Type': 'application/json', 'Content-Length': '42', 'Host': '127.0.0.1', 'User-Agent': 'Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:80.0) Gecko/20100101 Firefox/80.0', 'Accept': 'application/json, text/plain, */*', 'Accept-Language': 'en-US,en;q=0.5', 'Accept-Encoding': 'gzip, deflate', 'Origin': 'http://127.0.0.1', 'Connection': 'keep-alive', 'Referer': 'http://127.0.0.1/auth/login', 'Cookie': 'csrftoken=yMuneMLxwsjKfpMafVj3dvYYAKrD3TuCUWuvINYFbjIZTL2mpij3X29TmSlZaPND'}
2020-09-07 07:01:28 - api_app.auth - obtain_user_token - INFO - administrator:'root' was logged in.
2020-09-07 07:01:33 - api_app.api - get_analyzer_configs - INFO - get_analyzer_configs received request from root.
2020-09-07 07:01:43 - api_app.api - ask_analysis_availability - INFO - ask_analysis_availability received request from root.Data: {'md5': ['40ff44d9e619b17524bf3763204f9cbb'], 'analyzers_needed': ['Whoisxmlapi']}
2020-09-07 07:01:43 - api_app.api - send_analysis_request - INFO - send_analysis_request received request from root.Data:{'md5': '40ff44d9e619b17524bf3763204f9cbb', 'analyzers_requested': ['Whoisxmlapi'], 'run_all_available_analyzers': False, 'force_privacy': False, 'private': False, 'disable_external_analyzers': False, 'tags_id': [], 'is_sample': False, 'observable_name': '8.8.8.8', 'observable_classification': 'ip', 'runtime_configuration': {}}.
2020-09-07 07:01:43 - api_app.api - send_analysis_request - INFO - serialized_data: OrderedDict([('tags_id', []), ('is_sample', False), ('md5', '40ff44d9e619b17524bf3763204f9cbb'), ('observable_name', '8.8.8.8'), ('observable_classification', 'ip'), ('analyzers_requested', ['Whoisxmlapi']), ('run_all_available_analyzers', False), ('force_privacy', False), ('disable_external_analyzers', False), ('runtime_configuration', {})])
2020-09-07 07:01:43 - api_app.api - send_analysis_request - INFO - New Job added with ID: #1 and md5: 40ff44d9e619b17524bf3763204f9cbb.
2020-09-07 07:01:43 - api_app.script_analyzers.utils - set_job_status - INFO - setting job_id 1 to status running
2020-09-07 07:02:00 - api_app.crons - flush_expired_tokens - INFO - started flush_expired_tokens
2020-09-07 07:02:00 - api_app.crons - flush_expired_tokens - INFO - finished flush_expired_tokens
2020-09-07 07:03:00 - api_app.crons - flush_expired_tokens - INFO - started flush_expired_tokens
2020-09-07 07:03:00 - api_app.crons - flush_expired_tokens - INFO - finished flush_expired_tokens
2020-09-07 07:04:00 - api_app.crons - flush_expired_tokens - INFO - started flush_expired_tokens
2020-09-07 07:04:00 - api_app.crons - flush_expired_tokens - INFO - finished flush_expired_tokens
2020-09-07 07:05:00 - api_app.crons - check_stuck_analysis - INFO - started check_stuck_analysis
2020-09-07 07:05:00 - api_app.crons - flush_expired_tokens - INFO - started flush_expired_tokens
2020-09-07 07:05:00 - api_app.crons - check_stuck_analysis - INFO - checking if 1 jobs are stuck
2020-09-07 07:05:00 - api_app.crons - check_stuck_analysis - INFO - finished check_stuck_analysis
2020-09-07 07:05:00 - api_app.crons - flush_expired_tokens - INFO - finished flush_expired_tokens
```